### DISCUSSION TITLE
PIP-21: unbond token can rebond at any time

### DISCUSSION THREAD

**Discussion by ja7ad** [pactus-project/PIPs#90] [PIPs]
PIP-21: [unbond token can rebond at any time](https://pips.pactus.org/PIPs/pip-21)

**Comment by b00f** [comment]
User can get new high spec server and recover their wallet there . Then in this case they don't need to re-bond. Right?

**Reply by ja7ad** [reply]
@laosiji-io

**Reply by laosiji-io** [reply]
in simple terms, users can stake from their original validator address to a new validator address.
This needs to be a brand new node, with a verifier address that has not been staked before.

**Reply by b00f** [reply]
> in simple terms, users can stake from their original validator address to a new validator address.

We don't support stake move between validators, since there is a simple solution for them: recovering the node in new server.

**Reply by laosiji-io** [reply]
That's a pity, I think from the perspective of users, because if someone has a low score and wants to transfer to a better server, they may need to wait for 7 days to earn profits.

**Reply by b00f** [reply]
Yes. There is no shortcut to improve the `availability score`
